sys/net/gnrc/network_layer/sixlowpan/frag/sfr/gnrc_sixlowpan_frag_sfr.c

```diff
@@ -1209,9 +1209,11 @@ static void _check_failed_frags(sixlowpan_sfr_ack_t *ack,
 
 /* ====== INTERNAL FUNCTIONS USED BY PUBLIC FUNCTIONS ======
  * ====== AND TO MANIPULATE INTERNAL DATA STRUCTURES  ====== */
-static inline gnrc_sixlowpan_frag_sfr_bitmap_t *_to_bitmap(uint8_t *bitmap)
+static inline gnrc_sixlowpan_frag_sfr_bitmap_t *_get_bitmap(gnrc_sixlowpan_frag_rb_t *frag_rb)
```

Comment:
Ahh, so the bitmap in question was never checked for the header with this function :slightly_smiling_face: :woman_facepalming: .

Reply:
Yep. That is also why I lost the bet :-)